src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { kebabCase } from 'lodash/fp';
+import classNames from 'classnames';
+import Downshift from 'downshift';
+
+import { facilityTypes } from '../config';
+import { keyMap } from '../utils/helpers';
+
+const FACILITY_OPTIONS = ['all', 'health', 'benefits', 'cemetery', 'vet_center'];
+
+const facilityOptionClasses = (item, selected) => classNames(
+  'dropdown-option',
+  { selected },
+  { 'icon-option': item && item !== 'all' },
+  { [`${kebabCase(item)}-icon`]: item && item !== 'all' }
+);
+
+const itemToString = (item) => facilityTypes[item] || 'All Facilities';
+
+const focusDropdown = () => document.querySelector('#facility-dropdown-toggle').focus();
+
+const FacilityTypeDropdown = ({
+  getButtonProps,
+  getItemProps,
+  highlightedIndex,
+  isOpen,
+  selectedItem,
+  toggleMenu
+}) => {
+  const handleKeyDown = (e) => {
+    switch (e.keyCode) {
+      // Allow blurring focus (with TAB) to close dropdown.
+      case keyMap.TAB:
+        if (isOpen) { toggleMenu(); }
```

Comment:
Would've liked to move this event handler out so that it's not recreated on each render, but would've needed to pass `toggleMenu` and `isOpen` anyway.

Potential workarounds might make it less readable or difficult to follow, and there aren't too many options being rendered anyway.